docs/operator-manual/declarative-setup.md

```diff
@@ -682,6 +682,10 @@ Notes:
 * Invalid globs result in the whole rule being ignored.
 * If you add a rule that matches existing resources, these will appear in the interface as `OutOfSync`.
 
+## Resource Custom Labels
+
+Custom Labels configured with `resource.customLabels` (comma separated string) will be displayed in the UI (for any resource that defines them).
```

Comment:
Can you please add it to reference config map as well: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml